source/user-portal/assets/favRecipe.js

```diff
@@ -0,0 +1,108 @@
+class FavRecipe extends HTMLElement {
+    constructor() {
+      super();
+      this.attachShadow({mode: 'open'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8429335)
```suggestion
      this.attachShadow({mode: "open"});
```